From d03fdf41ec966eabcdf71323d389d301541c6cca Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Tue, 13 May 2014 21:58:31 +0200
Subject: [PATCH 08/14] MINOR: stats: improve alignment of color codes to save
 one line of header

Having both "active or backup DOWN" and "not checked" on the left side of
the color caption inflates the whole header block for no reason. Simply
move them both on the same line and reduce the header height.
---
 src/dumpstats.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/dumpstats.c b/src/dumpstats.c
index c76e115..eae736c 100644
--- a/src/dumpstats.c
+++ b/src/dumpstats.c
@@ -3846,7 +3846,6 @@ static void stats_dump_html_info(struct stream_interface *si, struct uri_auth *u
 	              "<td class=\"backup1\"></td><td class=\"noborder\">backup DOWN, going up </td>"
 	              "</tr><tr>\n"
 	              "<td class=\"active0\"></td><td class=\"noborder\">active or backup DOWN &nbsp;</td>"
-	              "</tr><tr>\n"
 	              "<td class=\"active8\"></td><td class=\"noborder\">not checked </td>"
 	              "</tr><tr>\n"
 	              "<td class=\"maintain\"></td><td class=\"noborder\" colspan=\"3\">active or backup DOWN for maintenance (MAINT) &nbsp;</td>"
-- 
1.8.3.2

